.github/workflows/test.yml

```
@@ -0,0 +1,45 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - "*"
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)